bluemira/base/parameter.py

```diff
@@ -1335,7 +1335,7 @@ def _from_iterable(value):
             and len(value) in [2, 3]
             and isinstance(value[1], (str, type(None)))
         ):
-            bluemira_debug("Consider using a dictionary for ordering safety")
+            # bluemira_debug("Consider using a dictionary for ordering safety")
```

Comment:
If this is being commented, can it be removed?

Reply:
I left it there as a note to any future developer but in principle yes, it can be removed

Reply:
A note like that I would probably put in the docstring, rather than in the form of commented code